### PR TITLE
Use new config data when Infection has been executed in a first time (without .json or json.dist)

### PR DESCRIPTION
**Pull request by sidz**
This PR fixes https://github.com/infection/infection/issues/576

### CODE REVIEW

File: src/Command/InfectionCommand.php [hunk 1]
```diff
@@ -352,10 +352,9 @@ private function runConfigurationCommand(Locator $locator): void
             $locator->locateOneOf(InfectionConfig::POSSIBLE_CONFIG_FILE_NAMES);
         } catch (\Exception $e) {
             $configureCommand = $this->getApplication()->find('configure');
-            $config = $this->getContainer()->get('infection.config');
 
             $args = [
-                '--test-framework' => $this->input->getOption('test-framework') ?: $config->getTestFramework(),
+                '--test-framework' => $this->input->getOption('test-framework') ?: TestFrameworkTypes::PHPUNIT,
```
Comment (sidz) [inline src/Command/InfectionCommand.php]: @borNfreee it's a hot fix which I may suggest so far.

File: src/Config/InfectionConfig.php [hunk 2]
```diff
@@ -35,6 +35,7 @@
 
 namespace Infection\Config;
 
+use _HumbugBox40a58e7ed96f\Infection\TestFramework\TestFrameworkTypes;
```
Comment (borNfreee) [inline src/Config/InfectionConfig.php]:
```suggestion
use Infection\TestFramework\TestFrameworkTypes;
```

**Comment by sidz** [comment]
> I agree with that guy, can we have a test to prevent regression?

@borNfreee Fine with me if we are agree to continue with this approach.

**Comment by borNfreee** [comment]
> sidz removed the in progress  label a day ago

something is wrong in Travis when `phpdbg` is used. I have restarted the builds but it doesn't help.

**Comment by sidz** [comment]
@borNfreee yeah I know. It smth what I broke in this PR but didn't find what exactly yet

**Comment by borNfreee** [comment]
ok, I thought it's ready because of removed "in progress" label ( ͡° ͜ʖ ͡°)

**Comment by borNfreee** [comment]
> had another one round of checking what is going on. Without success. Any help?

@sidz could you please target 0.12 branch? I will see how I can help to debug travis failures

**Comment by borNfreee** [comment]
Superseded by https://github.com/infection/infection/pull/651. 

Thank you @sidz

**Pull request closed without merging**